test/integration/smart_answer_flows/energy_grants_calculator_test.rb

```diff
@@ -55,6 +55,21 @@ class EnergyGrantsCalculatorTest < ActiveSupport::TestCase
             assert_current_node :outcome_help_with_bills # outcome 1
           end
         end
+
+        context "answer before 5th May 1953" do
+          setup do
+            add_response '01/05/1953'
+          end
+
+          should "be eligible for winter fuel payment" do
+            assert_equal :winter_fuel_payment, current_state.calculator.age_variant
+          end
```

Comment:
Not very readable, especially 'assert_equal :winter_fuel_payment, @calculator.age_variant', but Ok.